src/message.rs

```diff
@@ -29,7 +29,7 @@ impl Message {
     ///
     /// There is no guarantee the provided bytes will be the product of a cryptographically secure
     /// hash. Using insecure messages might compromise the security of the signature.
-    pub unsafe fn from_bytes_unchecked(bytes: [u8; Self::LEN]) -> Self {
+    pub fn from_bytes_unchecked(bytes: [u8; Self::LEN]) -> Self {
```

Comment:
Victor's goal with `unsafe` here was (I assume) to provide a warning to the users that you are responsible for what bytes you provide in here. While `unsafe` is definitely not the way to go, not sure is there anything else we can do to assert are the input bytes `cryptographically secure hash` @Voxelot ? or at least provide a comment "use at your own discretion" 😅